src/porepy/numerics/fv/biot.py

```diff
@@ -453,15 +405,131 @@ def build_rhs_normals_single_dimension(dim):
         bound_div_d = div * igrad * rhs_bound
         del rhs_cells
 
-        stabilization = div * igrad * rhs_normals
-
         matrices_m["stress"] = stress
         matrices_m["bound_stress"] = bound_stress
-        matrices_m["grad_p"] = grad_p
         matrices_m["div_d"] = div_d
-        matrices_f["biot_stabilization"] = stabilization
         matrices_m["bound_div_d"] = bound_div_d
 
+        # ----------------------------------------------------------------------------
+
+        # Biot discretization
+
+        # ----------------------------------------------------------------------------
+
+        # Take Biot's alpha as a tensor
+        alpha = parameters_m["biot_alpha"]
```

Comment:
biot_alpha or biot_coefficient? I think I lean towards the latter, but I am open for other opinions.